### PR TITLE
 Create an application blog and model for that

### PR DESCRIPTION
**Pull request by serhii73**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- repo: https://github.com/pre-commit/pre-commit-hooks
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: These are unrelated changes.

File: blog/admin.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
-from django.contrib import admin
+"""Settings for admin Blog app."""
```
Comment (webknjaz) [inline blog/admin.py]: it's more like configuration, not settings.

File: blog/migrations/0001_initial.py [hunk 3]
```diff
@@ -1,3 +1,4 @@
+"""Initial migrations."""
```
Comment (webknjaz) [inline blog/migrations/0001_initial.py]: you were supposed to not touch migrations as they are autogenerated.

File: blog/views.py [hunk 4]
```diff
@@ -1,3 +1,3 @@
-from django.shortcuts import render
+"""Business logic Blog app."""
```
Comment (webknjaz) [inline blog/views.py]: this is a nonsense set of words

Comment (serhii73) [reply blog/views.py]: @webknjaz What would you write?

File: blog/migrations/0001_initial.py (hunk 3 above)
Comment (serhii73) [reply blog/migrations/0001_initial.py]: @webknjaz Do I understand correctly? I need to add migrations to `.gitignore`? Right?

File: blog/views.py (hunk 4 above)
Comment (webknjaz) [reply blog/views.py]: Views for the blog app.

File: blog/migrations/0001_initial.py (hunk 3 above)
Comment (webknjaz) [reply blog/migrations/0001_initial.py]: no. you can exclude it in pre-commit tool

**Comment by webknjaz** [comment]
(commit messages are horrible though)

**Comment by serhii73** [comment]
@webknjaz I would be grateful if you said that was wrong, and how you wrote it.
